pkgs/top-level/all-packages.nix

```diff
@@ -9278,6 +9278,11 @@ in
     openssl = null;
   };
 
+  openconnect_head = callPackage ../tools/networking/openconnect {
+    head = true;
+    openssl = null;
```

Comment:
```suggestion
```

Reply:
Doing so would probably break the build, see the lines above.